### DISCUSSION TITLE
Replace dbx with Bun

### DISCUSSION THREAD

**Discussion by surendrakandel** [pocketbase/pocketbase#1383] [Polls]
https://github.com/uptrace/bun

Bun is minimal golang orm for sql. Its more actively worked on and has similar syntax as dbx. I think it will more project in right direction.

**Comment by ganigeorgiev** [comment]
But why? For the end user there aren't really much benefits doing it.

Additionally, at least the last time I checked, bun didn't support structured where expressions (aka. combining and building multiple expressions outside of the query context, eg. `dbx.And(dbx.Like(), dbx.In(), ...)`).

**Reply by dschissler** [reply]
@ganigeorgiev I really like [goqu](https://github.com/doug-martin/goqu).  It has powerful expression structs.

**Reply by ganigeorgiev** [reply]
@dschissler I also like goqu, I've used it in the past, but it is way more complex than dbx.
dbx is fairly small (10-15 go files), its source is easy to read/understand and I feel confident that I can maintain it without much troubles.

In any case, there are no plans to switch the db lib at the moment.

**Reply by surendrakandel** [reply]
thank you.It make sense. I was thinking in terms of more of use it as framework.
